datasets/biwi_kinect_head_pose/README.md

```diff
@@ -0,0 +1,304 @@
+            {
+                'rotation': [[1.0, 0.0, 0.0], [0.0, 1.0, 0.0], [0.0, 0.0, 1.0]], 
+                'translation': [0.0, 0.0, 0.0]
+            }
+        }
+    'head_pose_gt': 
+        {
+            'center': [[43.4019, -30.7038, 906.864], [43.0202, -30.8683, 906.94], [43.0255, -30.5611, 906.659], .....],
+            'rotation': [[[0.980639, 0.109899, 0.162077], [-0.11023, 0.993882, -0.00697376], [-0.161851, -0.011027, 0.986754]], ......]
+        }
+}
+```
+
+### Data Fields
+
+- `sequence_number` : This refers to the sequence number in the dataset. There are a total of 24 sequences.
+- `subject_id` : This refers to the subjects in the dataset. There are a total of 20 people with 6 females and 14 males where 4 people were recorded twice.
+- `rgb` : List of png frames containing the poses.
+- `rgb_cal`: Contains calibration information for the color camera which includes intrinsic matrix, 
+global rotation and translation.
+- `depth` : List of depth frames for the poses.
+- `depth_cal`: Contains calibration information for the depth camera which includes intrinsic matrix, global rotation and translation.
+- `head_pose_gt` : Contains ground truth information, i.e., the location of the center of the head in 3D and the head rotation, encoded as a 3x3 rotation matrix.
+
+
+### Data Splits
+
+All the data is contained in the training set.
+
+## Dataset Creation
+
+### Curation Rationale
+
+[More Information Needed]
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+The Biwi Kinect Head Pose Database is acquired with the Microsoft Kinect sensor, a structured IR light device.
+#### Who are the source language producers?
+
+[More Information Needed]
+
+### Annotations
+
+#### Annotation process
+
+From Dataset's README : 
+> The database contains 24 sequences acquired with a Kinect sensor. 20 people (some were recorded twice - 6 women and 14 men) were recorded while turning their heads, sitting in front of the sensor, at roughly one meter of distance.
+
+#### Who are the annotators?
+
+[More Information Needed]
+
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[More Information Needed]
+
+### Discussion of Biases
+
+[More Information Needed]
+
+### Other Known Limitations
+
+[More Information Needed]
+
+## Additional Information
+
+### Dataset Curators
+
+[Needs More Information]
+
+### Licensing Information
+
+From Dataset's README : 
+> This database is made available for non-commercial use such as university research and education.
+
+### Citation Information
+
+```bibtex
+@article{fanelli_IJCV,
+  author = {Fanelli, Gabriele and Dantone, Matthias and Gall, Juergen and Fossati, Andrea and Van Gool, Luc},
+  title = {Random Forests for Real Time 3D Face Analysis},
+  journal = {Int. J. Comput. Vision},
+  year = {2013},
+  month = {February},
+  volume = {101},
+  number = {3},
+  pages = {437--458}
+}
+```
```

Comment:
````suggestion
```

### Contributions

Thanks to [@dnaveenr](https://github.com/dnaveenr) for adding this dataset.
````